packages/calypso-e2e/src/data-helper.ts

```diff
@@ -167,6 +167,15 @@ export function getTosUploadToken(): string {
 	return bearerToken;
 }
 
+/**
+ * Returns the site upload destination for the ToS screenshots.
+ *
+ * @returns {string} Site ID of the destination to which uploaded.
+ */
+export function getTosUploadDestination(): string {
+	return '200900774'; // wpcom site ID
```

Comment:
I think this should be included in the encrypted secrets file if possible, but there's an active PR (https://github.com/Automattic/wp-calypso/pull/63183) to refactor the entire secrets file. 

At this moment, I think we can keep the raw siteID in code, but as soon as the other PR lands I will look into moving this into the secrets.

Reply:
Thanks @worldomonation . If the PR otherwise looks good, could you please approve it?